app/components/questions/BelongingRates.js

```diff
@@ -61,6 +60,9 @@ function transformData(response, error, selectedArea) {
 function ExamplePieChart(props) {
   const { data } = props
   
+  if(!data[0].value){
```

Comment:
The automated test for the website build is failing with this error message:

```
Error occurred prerendering page "/questions/belonging-example". Read more: https://nextjs.org/docs/messages/prerender-error
TypeError: Cannot read property 'value' of undefined
    at BelongingRates_ExamplePieChart (/home/runner/work/transithealth/transithealth/app/build/server/pages/questions.js:4075:16)
```

I think it is referring to line 63 in your change. This could be a good time to use optional chaining:

```js
if (!data?.[0]?.value) {
  return null;
}
```

Now, if `data[0]` is `undefined`, the optional chaining will evaluate early to `undefined` instead of trying to check `undefined.value` and causing an error.

Try that and let me know if it both fixes the site build and achieves your intended effect for areas that have values vs Burnside.

Reply:
This seems to work now, thanks for the help.